library/psa_crypto_storage.c

```diff
@@ -96,12 +96,13 @@ static psa_status_t psa_crypto_storage_load( const psa_key_file_id_t key,
     psa_status_t status;
     psa_storage_uid_t data_identifier = psa_its_identifier_of_slot( key );
     struct psa_storage_info_t data_identifier_info;
+    size_t data_length = 0;
 
     status = psa_its_get_info( data_identifier, &data_identifier_info );
     if( status  != PSA_SUCCESS )
         return( status );
 
-    status = psa_its_get( data_identifier, 0, (uint32_t) data_size, data );
+    status = psa_its_get( data_identifier, 0, (uint32_t) data_size, data, &data_length );
```

Comment:
Should we check `data_length` is equal to `data_size` and return an error (probably `PSA_ERROR_STORAGE_FAILURE`) if not?

Reply:
yes, its better.